articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -479,6 +479,7 @@ From the Azure CLI, follow the steps below. If you don't have the Azure CLI inst
     | `--destination`        | Enter `appservice` to indicate that you're exporting data to App Service.                                                                                          | `appservice`                                                                                                              |
     | `--appservice-account` | Enter the App Service's ARM ID or use the name of the App Service, if it's in the same subscription and resource group as the App Configuration.                   | `/subscriptions/123/resourceGroups/my-as-resource-group/providers/Microsoft.Web/sites/my-app-service` or `my-app-service` |
     | `--label`              | Enter a label to export keys and feature flags with this label. If you don't specify a label, by default, you will only export keys and feature flags with no label. You can enter one label, enter several labels by separating them with `,`, or use `*` to take all of the labels in account. | `prod`                                                                                                                    |
```

Comment:
I don't think we support exporting more than one label to App Service. Can you please delete the sentence below?

"You can enter one label, enter several labels by separating them with `,`, or use `*` to take all of the labels in account."